packages/lib/services/plugins/utils/types.ts

```diff
@@ -7,6 +7,13 @@ export interface Screenshot {
 	label: string;
 }
 
+export interface Icons {
+	16: string;
+	32: string;
+	48: string;
+	128: string;
+}
```

Comment:
Let's do like Firefox does - there should be at least one icon but all the other ones are optional. So it means all fields are optional.